src/asInput/index.jsx

```diff
@@ -50,8 +53,17 @@ const asInput = (WrappedComponent, labelFirst = true) => {
 
       if (!this.state.isValid) {
         desc.error = (
-          <div className={styles['form-control-feedback']} id={errorId} key="0">
-            {this.state.validationMessage}
+
+          <div className={classNames(styles['form-control-feedback'], { [styles['invalid-feedback']]: this.props.theme.includes('danger') })} id={errorId} key="0">
+            { this.props.theme.includes('danger') &&
+            <span
```

Comment:
This is an alert icon, which follows UI/UX guidelines laid out [here](https://openedx.atlassian.net/wiki/spaces/UX/pages/156737282/Forms).